client/src/pages/Profile.js

```diff
@@ -212,17 +193,21 @@ const Profile = ({ location }) => {
                 </div>
                 <div className={style.activePanel}>
                     <div className={style.gridContainer}>
-                        <Card className={style.card}>
-                            <CardActionArea className={style.card}>
-                                <CardMedia className={style.cardImg} image={house} />
-                                <Typography variant='h6' className={style.cardHeader}>
-                                    House
-                                </Typography>
-                                <Typography variant='p' className={style.cardHeader}>
-                                    80 posts
-                                </Typography>
-                            </CardActionArea>
-                        </Card>
+                            {
+                                posts.map((post, i) => {
+                                    return <Card className={style.card}>
+                                    <CardActionArea className={style.card}>
+                                        <CardMedia className={style.cardImg} image={house} />
+                                        <Typography variant='h6' className={style.cardHeader}>
+                                            {post}
+                                        </Typography>
+                                        <Typography variant='p' className={style.cardHeader}>
+                                            80 posts
+                                        </Typography>
+                                    </CardActionArea>
+                                </Card>
+                                })
+                            }
```

Comment:
you probably want to show something if there are no posts or it's loading etc.

Reply:
got it